jenkins/post_comment.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -ex
+
+ERRORS_FILENAME=$WORKSPACE/erros.log
+
+# This script is kicked with $1=0 
+# only if "bash build_and_test.sh" has exited successfully.
+if [[ $1 -eq 0 ]]; then
+  BODY="Test(${BUILD_DISPLAY_NAME}, ${GIT_COMMIT}) passed in $(uname -n)."
```

Comment:
What is `${BUILD_DISPLAY_NAME}`? Do not show this if it corresponds to jenkins server job id because Jenkins server is not public.

Reply:
The rendered result that `#NN` has a link to an issue is unexpected for me.

I intended to navigate to the detailed build result (on the in-house server) by displaying `${BUILD_DISPLAY_NAME}`.  
(If there isn't, the relationship between a comment (by maekawa) and the build result is not clear.)

By the way, I'll delete them.